Osiris/SDK/NetworkChannel.h

```diff
@@ -7,5 +7,16 @@ class NetworkChannel {
     VIRTUAL_METHOD(float, getLatency, 9, (int flow), (this, flow))
 
     std::byte pad[44];
+    int OutSequenceNr;
+    int InSequenceNr;
+    int OutSequenceNrAck;
+    int OutReliableState;
+    int InReliableState;
```

Comment:
nice paste XD  
If you want to add 5 int's here you have to deduct `5*sizeof(int)` from the padding above.  
Have you even tested this before making a pr?

Reply:
i know its a paste LOL do you think ill waste my time and selfcode it when theres a source with it? its literally called "based on" so i literally pasted it you dumb fuck, and i wont go to uc and ask anyone if im about to fucking paste you dumb fucking moron. what a monkey

Reply:
lmao what a joke

Reply:
it actually works lmfao

Reply:
These monkeys keep complaining on a clearly pasted pr but won't make a new pr on a better sv_pure bypass yikes

Reply:
> These monkeys keep complaining on a clearly pasted pr but won't make a new pr on a better sv_pure bypass yikes

true